src/app/shared/components/funding-progress/funding-progress.component.html

```diff
@@ -1,52 +1,63 @@
-<ng-container *ngIf="(progressData$ | async) as data">
-  <ng-container *ngIf="campaign">
-    <div *ngIf="data.raised < data.total && data.total >= data.softCap">
-      <span class="mt-6 font-bold text-gray-800 text-sm">{{ data.raised | currencyDefault }}</span>
-      <span class="mx-1 text-sm">of</span>
-      <span class="mt-6 font-bold text-gray-800 text-sm">{{ data.softCap | currencyDefault }}</span>
-      <div class="relative h-2 mb-2 pt-1 mt-2">
-        <!-- Progress tray -->
-        <div class="absolute w-full h-full rounded bg-gray-200"></div>
+<div *ngIf="(progressData$ | async) as dataRes" class="flex items-center justify-center min-h-[5rem]">
```

Comment:
Added loading spinner and error state for funding progress, as well as minimum height to keep things consistent. Ignore the red border in the images, it is here only to show the actual component borders and has been removed from the app.

# Ready state
![Screenshot 2021-10-04 at 14 55 59](https://user-images.githubusercontent.com/12549625/135855669-9d1fa495-9162-459c-bcde-0be331f848e6.png)

# Loading state
![Screenshot 2021-10-04 at 14 51 25](https://user-images.githubusercontent.com/12549625/135855724-15e59b03-9687-4b35-a52c-54f5a55260d1.png)
![Screenshot 2021-10-04 at 14 55 32](https://user-images.githubusercontent.com/12549625/135855743-4c397596-f4a1-4f92-9fc5-c35827430125.png)

# Error state
![Screenshot 2021-10-04 at 14 54 27](https://user-images.githubusercontent.com/12549625/135855794-914776ce-821a-4015-b8f1-359ab7265587.png)
![Screenshot 2021-10-04 at 14 54 57](https://user-images.githubusercontent.com/12549625/135855799-b034a8e7-36f0-4638-83f0-7268770d1f00.png)